.github/workflows/publish.yml

```diff
@@ -0,0 +1,29 @@
+name: Generate a build and push to another dist branch
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Build and Push
+    steps:
+      - name: git-checkout
+        uses: actions/checkout@v2
+
+      - name: Install all dependencies
+        run: npm install
+
+      - name: Build
+        run: npm run build
+
+      - name: Push
```

Comment:
As I understand it, this will ONLY publish the `dist` directory. I think the behaviour we want right now (which would mimic the behaviour used for NPM), is that we want to include the `dist` directory along with everything else, and push that to the `dist` branch.

So, I recommend using https://github.com/marketplace/actions/add-commit. We use that when update the collection stats every night on laws.africa [see here](https://github.com/laws-africa/laws-africa.github.io/blob/master/.github/workflows/update-stats.yml)